CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 
 Changes that are currently in development and have not been released yet.
 
+
+## [0.14.0](https://github.com/cossacklabs/themis/releases/tag/0.14.0), December 24th 2021
+
+**TL;DR:**
+
+- JavaThemis for Android has moved to Maven Central ([read more](#0.14-maven-central)).
+- JavaThemis for Java is available on Maven Central ([read more](#0.14-maven-central)).
+- TypeScript definitions and ES6 module are now available for WasmThemis ([read more](#0.14-typescript)).
+- Node.js v16 is now supported.
```

Comment:
```suggestion
- Node.js v16 is now supported.
- Themis for iOS is available through SPM, Cocoapods and Carthage. Uh.
```

Reply:
Damn, I forgot that SPM support was not a part of 0.13.0, but appeared along the way. Now I gotta review the changes in the patch versions...